local-cli/templates/HelloWorld/_flowconfig

```diff
@@ -2,6 +2,15 @@
 ; We fork some components by platform
 .*/*[.]android.js
 
+; Ignore templates for 'react-native init'
+.*/local-cli/templates/.*
+
+; Ignore the website subdir
+<PROJECT_ROOT>/website/.*
+
+; Ignore the Dangerfile
+<PROJECT_ROOT>/danger/dangerfile.js
```

Comment:
these are probably not required or we should update the path to node_modules/react-native/danger ...